app/pages/project/classify.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+
+import { FakeApiClient } from '../../../test/fake-api-client';
+import ProjectClassifyPage from './classify';
+import FinishedBanner from './finished-banner';
+
+describe('ProjectClassifyPage', function () {
```

Comment:
Don't be too fooled by these, since all the interesting bits require API interaction to load workflows, subjects etc, and that's really not tested afaik?